apps/server/src/modules/learnroom/learnroom.module.ts

```diff
@@ -45,6 +46,7 @@ import {
 		CourseGroupService,
 		CourseGroupRepo,
 		DashboardService,
+		CommonCartridgeMapper,
```

Comment:
why here? shouldn't it be the reverse cartdige knows learn room but not the other way around?

Reply:
@Loki-Afro No, it is correct the way it is. It is mapped here from course to CC. CC is the goal.